rllib/agents/trainer.py

```diff
@@ -749,8 +749,9 @@ def env_creator_from_classpath(env_context):
         # Evaluation setup.
         self.evaluation_workers = None
         self.evaluation_metrics = {}
-        # Do automatic evaluation from time to time.
-        if self.config.get("evaluation_interval"):
+        # User would like to setup a separate evaluation worker set.
```

Comment:
Small change in the code to make the new example (RLlib in 60s) work with fewer config-overrides.